src/test/tests/databases/plaintext.py

```diff
@@ -117,7 +117,9 @@ def TestCSVCurves():
     CreateCurvesDataFile("curves.csv")
 
     # Curve plot {
-    plainTextOpenOptions = GetDefaultOpenOptions()
+    # GetDefaultOpenOptions is a wrapper to the CLI method GetDefaultFileOpenOptions.
+    # GetDefaultOpenOptions function deepcopy to ensure we always start from a prestine default state for open options.
```

Comment:
```suggestion
    # GetDefaultOpenOptions() is a wrapper internal to our tests to the CLI method GetDefaultFileOpenOptions().
    # It ensures we always start from a pristine default state for open options.
```

Reply:
And I would adjust this in the other two places as well.

Reply:
Apologies for multiple PRs... I saw your comments a bit late.